### DISCUSSION TITLE
【求助】通过watchtower升级后，部分数据丢失。

### DISCUSSION THREAD

**Discussion by gba3455** [whyour/qinglong#1759] [Q&A]
大佬，你好。
我是从v2.12.2通过
docker run --rm   -v /var/run/docker.sock:/var/run/docker.sock   containrrr/watchtower -cR  ql
这个命令直接v2.15.4
升级后可以正常进入面板。
就是定时任务，环境变量，依赖管理，系统设置-应用设置，系统设置-通知设置，系统设置-登录日志里的数据都没了。
但是脚本管理里还可以看到之前的所有脚本。
配置文件里的配置内容也是之前旧版的内容。
请问大佬，怎么才能恢复这些配置呢？

**Comment by gba3455** [comment]
> 大佬，你好。 我是从v2.12.2通过 docker run --rm -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower -cR ql 这个命令直接v2.15.4 升级后可以正常进入面板。 就是定时任务，环境变量，依赖管理，系统设置-应用设置，系统设置-通知设置，系统设置-登录日志里的数据都没了。 但是脚本管理里还可以看到之前的所有脚本。 配置文件里的配置内容也是之前旧版的内容。 请问大佬，怎么才能恢复这些配置呢？

我自己搞定了。。

**Reply by blue-ice-layer** [reply]
怎么解决的，我也遇到同样的问题

**Reply by gba3455** [reply]
从旧版里爸数据备份出来，拷贝到新版里，刷新就走了
